examples/creatives/nonlinear_overlay.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * A sample SIMID nonlinear ad that shows an overlay of
+ * buttons to showcase functionality
+ */
+class NonLinear extends BaseSimidCreative {
+    constructor() {
+      super();
+
+      this.addButtonClickActions_();
+    }
+
+    /**
+     * Adds actions to different buttons available on the overlay.
+     * @private
+     */
+    addButtonClickActions_() {
+      this.sendMessageOnButtonClick_("close_ad", CreativeMessage.REQUEST_STOP);
+    }
+
+    onRequestResize() {
```

Comment:
A lot of this code is mixed with code I've looked at in another PR.  Ideally this PR should only contain new changes.

Reply:
The reason for this is because I branched off of Kaleab's work but he didn't yet merge it into master